io.github.go_musicfox.go_musicfox.yml

```diff
@@ -0,0 +1,35 @@
+id: io.github.go_musicfox.go_musicfox
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.golang
+command: go-musicfox
+
+finish-args:
+    - --share=network
+    - --socket=pulseaudio
+    - --talk-name=org.freedesktop.Notifications
```

Comment:
what do you use for notifications? libnotify?

Reply:
Yes. I'm not the author, but after tracing through the function calls, notifications are sent by executing `notify-send` using a fork of [notificator](https://github.com/0xAX/notificator).

Reply:
Shelling out to a cli tool to send notifications is weird when cross-platform APIs exist for the last 10-15 years.

Anyway, notify-send uses the portal but I do not know about kdialog.

Reply:
and multiple go bindings for libnotify exists.

Reply:
kdialogue doesn't use the portal, but anyway, if it used the APIs directly it wouldn't have been needed.

Reply:
The part about `kdialog` in the README file seems to be false advertising. The notification library only mentions it once in the README and doesn't call `kdialog` anywhere in the code. I agree that it is not ideal to rely on sketchy, unmaintained libraries, but migrating to some other notification library upstream would depend on how the maintainer thinks about it.